ssn/chapters/Common.html

```diff
@@ -70,17 +70,17 @@ <h6><a href="#SOSAFeatureOfInterest">sosa:FeatureOfInterest</a></h6>
             </p>
             <p>
                 <em>Feature Of Interest</em> &mdash;
-                The thing that is the target of an <a href="#SOSAExecution">Execution</a> or
-                <a href="#SOSADeployment">Deployment</a>,
+                thing that is the target of an Execution or
```

Comment:
A "The" disappeared from the beginning of the sentence. Ignore if intentional.

Reply:
Looks unintentional to me.
```suggestion
                The thing that is the target of an Execution or
```

Reply:
Presence and type of article ('a' or 'the') was inconsistent, and did not seem to add anything to the meaning of the definition, so I removed them all to be consistent. 

I followed ISO conventions by removing the term itself from the definition, and starting with lower-case.